src/ipc-types.d.ts

```diff
@@ -34,7 +34,6 @@ interface StoppedSiteDetails {
 interface StartedSiteDetails extends StoppedSiteDetails {
 	running: true;
 
-	port: number;
```

Comment:
Missed removing it in [this my PR](https://github.com/Automattic/studio/pull/944), so decided to remove in this PR